code/_onclick/hud/action.dm

```diff
@@ -90,8 +87,8 @@
 		current_button.overlays += img
 
 /obj/screen/movable/action_button
-	var/datum/action/owner
-	screen_loc = "WEST,NORTH"
+	var/datum/action/linked_action
```

Comment:
renamed to linked_action since "owner" is confusing, especially since /datum/action has its own owner var that references a mob.